core/src/main/java/org/everit/json/schema/loader/JsonValue.java

```diff
@@ -234,4 +234,17 @@ public Integer requireInteger() {
         }
         throw ls.createSchemaException(typeOfValue(), Integer.class);
     }
+
+    public static Object deepToOrgJson(JsonValue v) {
```

Comment:
Please make this package-private. I can see how is this necessary for now, but it is a bit redundant with `JsonValue#value()` and `JsonValue#unwrap()` (and their overrides in `JsonObject` and `JsonArray`) so this should be cleaned up in the future (and if you make this `public` then it can't be deleted in any 1.x versions).